uwp/Pdf-Viewer/Concepts-and-Features/Working-with-free-text-annotation.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Free text annotation in UWP PDF Viewer control | Syncfusion
+description: Learn here all about free text annotation support in Syncfusion UWP PDF Viewer (SfPdfViewer) control and more.
+platform: UWP
+control: SfPdfViewerControl
+documentation: ug
+---
+
+# Working with free text annotations in UWP PDF Viewer (SfPdfViewer)
+
+PDF Viewer allows you to include free text annotations in a PDF document and provides options to modify or remove the existing free text annotations.
+
+## Adding free text annotations
+
+### Enabling free text annotation mode
+
+To enable a free text annotation, execute the `FreeTextAnnotationCommand` with true as parameter.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfPdfViewerControl x:Name="pdfViewer"/>
+<Button x:Name="FreeTextAnnotationButton" Click="FreeTextAnnotationButton_Click"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
+private void FreeTextAnnotationButton_Click(object sender, RoutedEventArgs e)
+{
+	pdfViewer.FreeTextAnnotationCommand.Execute(true);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Disabling free text annotation mode
+
+Free text annotation can be disabled by executing the same command with false as parameter.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfPdfViewerControl x:Name="pdfViewer"/>
+<Button x:Name="resetAnnotationButton" Click="resetAnnotationButton_Click" />
+
+{% endhighlight %}
+{% highlight c# %}
+
+private void resetAnnotationButton_Click(object sender, RoutedEventArgs e)
+{
+	pdfViewer.FreeTextAnnotationCommand.Execute(false);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+##Customizing the appearance of free text annotations
+
+You can customize the default values of text color, text size, stroke color, fill color and stroke width of all free text annotations to be added. This will not affect the already added free text annotations.
+
+###Setting the default text color
+
+You can set the default text color of the free text annotations by using the `SfPdfViewerControl.FreeTextAnnotationSettings.TextColor` property. Refer to the following code.
```

Comment:
<img width="380" alt="image" src="https://github.com/syncfusion-content/uwp-docs/assets/65169077/eb894da1-1456-415a-89f7-ee25b8f3cbea">

Instead of nesting like this, can we mention as `TextColor` property of the `FreeTextAnnotationSettings` of `SfPdfViewerControl`

Reply:
Yes, we can use your suggestion but I have followed the same content structure used in the other annotations.
https://help.syncfusion.com/uwp/pdf-viewer/concepts-and-features/working-with-ink-annotation#setting-the-default-stroke-color

Shall I change this or leave it the same?

Reply:
Feedback addressed.